deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: local-storage-operator
       containers:
         - name: local-storage-operator
-          image: quay.io/gnufied/local-storage-operator:v0.0.8
+          image: quay.io/gnufied/local-storage-operator:v0.0.13
```

Comment:
still no "official" image?

Reply:
Updated these to more official OKD images.